components/policy/resources/templates/policy_definitions/BraveSoftware/TorDisabled.yaml

```diff
@@ -0,0 +1,39 @@
+caption: Disable Tor functionality
+default: null
+desc: |-
+  Disable Tor functionality in Brave.
+
+  Using Tor makes it more difficult to trace a user's Internet activity by preventing any single point on the Internet (other than the user's device) from being able to view both where traffic originated from and where it is ultimately going to at the same time. This conceals a user's location and usage from anyone performing network surveillance or traffic analysis from any such point, protecting the user's freedom and ability to communicate confidentially.
```

Comment:
cc: @diracdeltas @fmarier on the wording here! I copied it from Wikipedia 😱 Maybe we have a better summary that we should show. It shows in the Windows group policy editor which looks like this:
![image](https://github.com/user-attachments/assets/f1b67475-a9fc-4459-990d-1bf1eb2ca6ba)

Reply:
Seems good to me, but @diracdeltas may be aware of past discussions we've had with the Tor team about how we describe what Tor does.

Reply:
I would just copy the text we already have in Private Tabs; change title to "Disable Tor Connectivity" and change the text to:

Disable Tor connectivity in Brave.

Using Tor connectivity in Private Windows makes it more difficult for sites to see a user's true IP address and for network observers to see what sites they visit. This may be used for enhanced confidentiality and censorship resistance in Private Windows.

If this policy is set to true, Tor connectivity will always be disabled.

If this policy is set to false, Tor connectivity will be enabled if the user opens a Private Window with Tor. 

If you set this policy, users cannot change or override it.

If this policy is left unset, Tor connectivity will be enabled if the user opens a Private Window with Tor.

Reply:
Updated! 😄 Thanks